hw/dv/sv/i2c_agent/i2c_agent.core

```diff
@@ -9,7 +9,10 @@ filesets:
     depend:
       - lowrisc:dv:dv_utils
       - lowrisc:dv:dv_lib
+      - lowrisc:ip:i2c
```

Comment:
We should probably bring the constants / enums up to the agent's packages. It's kind of backwards for the agent to depend on the DUT.

Then the DV environment for the IP can reference the agent's packages for those values, if needed. Meanwhile, the RTL will continue to do its own thing.

If we wanted to create shared enums for even the RTL to use, I guess the place for that would be a core in the `prim` core library (prim_i2c_pkg? haha), since it's kind of our only common library.

Reply:
Yeah I felt like the RTL's package was the lowest common denominator for shared enums from what we had. A standalone prim pkg is probably a better abstraction though! I'll split it out.